README.md

```diff
@@ -3,42 +3,72 @@
 # React on Rails
 React on Rails integrates Facebook's [React](https://github.com/facebook/react) front-end framework with Rails. Currently, both React v0.14 and v0.13 are supported. See the Rails on Maui [blog post](http://www.railsonmaui.com/blog/2014/10/03/integrating-webpack-and-the-es6-transpiler-into-an-existing-rails-project/) that started it all!
```

Comment:
Be sure to see the [React Webpack Rails Tutorial Code](link) along with the live example at [www.reactrails.com](http://www.reactrails.com).